src/components/z-skip-to-content/index.tsx

```diff
@@ -0,0 +1,109 @@
+import {
+  Component,
+  h,
+  Host,
+  Element,
+  Listen,
+  Prop,
+  State,
+} from "@stencil/core";
+import { getElementTree, handleKeyboardSubmit } from "../../utils/utils";
+import { ThemeVariant, linkItem } from "../../beans";
+
+/**
+ * Component short description.
+ */
+@Component({
+  tag: "z-skip-to-content",
+  styleUrl: "styles.css",
+  shadow: false,
+  scoped: true,
+})
+export class ZSkipToContent {
+  /** Graphical variant: `dark`, `light`. */
+  @Prop({ reflect: true }) variant?: ThemeVariant;
+  /** Array to fill link into skip-content */
+  @Prop({ mutable: true }) links: string | linkItem[] = [];
+
+  @State() visible: boolean = true;
+  @State() visibleLink: string = "";
+
+  @Element() hostElement: HTMLElement;
+
+  @Listen("focusout", { target: "document" })
```

Comment:
Facendo così credo che faccia document.addEventListener("focusout", ecc...), siamo sicuri che questa cosa non possa generare un proliferare di listener se il componente si smonta e rimonta?

Reply:
:eyes:  [clicca-qui](https://stenciljs.com/docs/events#:~:text=The%20event%20listeners%20are%20automatically%20added%20and%20removed%20when%20the%20component%20gets%20added%20or%20removed%20from%20the%20DOM.)